readthedocs/proxito/tests/responses/v1.json

```diff
@@ -121,6 +121,9 @@
     }
   },
   "addons": {
+    "configs": {
+      "load_when_embedded": false
+    },
```

Comment:
.com tests will probably need to be updated

Reply:
Hrm, I don't think we have this file in .com. At least, I didn't find it

Reply:
We have two files in tests/responses/addons/